integrations/airflow/tests/extractors/test_bigquery_extractor.py

```diff
@@ -97,8 +97,10 @@ def test_extract(self, mock_client, mock_hook):
             'bigquery-public-data.usa_names.usa_1910_2013'
 
         assert step_meta.inputs[0].fields is not None
+        assert step_meta.inputs[0].source.name == \
+               'bigquery:'
```

Comment:
Is `bigquery:` a valid connection string for BigQuery?

Reply:
Sources conflict with that. For example, [rest API docs](https://cloud.google.com/bigquery/docs/reference/rest) use https://bigquery.googleapis.com. On the other hand, "official 3rd party" JDBC driver uses `https://www.googleapis.com/bigquery/v2`. On the third hand, official client libraries do not use concept of connection string, since there is one, global bigquery instance, and only thing that needs to be sorted is authentication.

Reply:
Thanks for the explanation, @mobuchowski!